tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -59,10 +59,14 @@ else
   KOPS="${KOPS_A}"
 fi
 
+
+
 ${KUBETEST2} \
 		--up \
 		--kops-binary-path="${KOPS_A}" \
 		--kubernetes-version="${K8S_VERSION_A}" \
+    --control-plane-size="${KOPS_CONTROL_PLANE_SIZE:-1}" \
+    --template-path="${KOPS_TEMPLATE:-}" \
```

Comment:
looks like this file uses tabs, not spaces

Reply:
The file uses a bit of both. I added a cleanup commit at the beginning of the branch now.